### ISSUE TITLE
Bareword "U" not allowed while "strict subs" in use

### ISSUE THREAD

**Issue by eserte** [sysread/URI-Fast#3]
Some of my smoker systems show the following failure:
```
Bareword "U" not allowed while "strict subs" in use at t/basics.t line 32.
Bareword "U" not allowed while "strict subs" in use at t/basics.t line 32.
Bareword "U" not allowed while "strict subs" in use at t/basics.t line 32.
Bareword "U" not allowed while "strict subs" in use at t/basics.t line 32.
Bareword "U" not allowed while "strict subs" in use at t/basics.t line 32.
Bareword "U" not allowed while "strict subs" in use at t/basics.t line 32.
Bareword "U" not allowed while "strict subs" in use at t/basics.t line 32.
Bareword "U" not allowed while "strict subs" in use at t/basics.t line 32.
Bareword "U" not allowed while "strict subs" in use at t/basics.t line 32.
Bareword "U" not allowed while "strict subs" in use at t/basics.t line 32.
Type of arg 1 to Test2::Tools::Exception::dies must be block or sub {} (not reference constructor) at t/basics.t line 178, near "},"
t/basics.t has too many errors.
One or more DATA sections were not processed by Inline.
```

The failure goes away if a recent Test2-Suite is installed.

**Comment by sysread** [comment]
Thank you for the report. I checked Test2::Suite's change log this morning and bumped the required version to 0.000049, which was the release when U was added to Test2::Bundle::Extended. This is fixed as of 78ffaa4b71342926bb216afcb8f09fd3bcb67b81. I'll push a new version to cpan shortly.

**Comment by sysread** [comment]
Fixed in v0.05.
